tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/identifying_field_suppression_test.py

```diff
@@ -155,8 +155,8 @@ def test_identifying_field_suppression(self):
                     'measurement_type_concept_id', 'provider_id'
                 ],
                 'loaded_ids': [321, 123],
-                'cleaned_values': [(321, 12345, 111, 444, None),
-                                   (123, 6789, 222, 555, None)]
+                'cleaned_values': [(321, 12345, 111, 444, 0),
+                                   (123, 6789, 222, 555, 0)]
```

Comment:
Why change these from `None` (which evaluates like `null` in the response) to 0?  `provider_id` is a nullable field.

Reply:
Typing issues.  The tests failed as the value was not compatible when set to 0.